plugin/src/main/java/io/jenkins/plugins/casc/impl/configurators/DataBoundConfigurator.java

```diff
@@ -241,7 +240,7 @@ public CNode describe(T instance, ConfigurationContext context) throws Exception
             Object value = a.getValue(instance);
             if (value != null) {
                 Object converted = Stapler.CONVERT_UTILS.convert(value, a.getType());
-                if (converted instanceof DescribableList || !a.isMultiple()) {
+                if (converted instanceof List || !a.isMultiple()) {
```

Comment:
Not sure I understand the conditions any longer :confounded: 

This is a list: https://github.com/jenkinsci/role-strategy-plugin/blob/40bb8d2afe3d74d725520b712339f37de135acf9/src/main/java/org/jenkinsci/plugins/rolestrategy/casc/GrantedRoles.java#L30

I guess it shows up as an `isMultiple` because of databound configurator?

Reply:
Stapler converts it to a string in the role strategy case but the attribute is still multiple

Reply:
Now I am sad :sob: